.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "redmine/plugins/redmine_text_blocks"]
```

Comment:
@dkastl (CC: @aruneko)  
Oh, I noticed that git submodule tag is not updated...  
I will fix this now.